charts/csi-cloudscale/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: cloudscale-csi
```

Comment:
Can we change this to `csi-cloudscale`? Or is it intentionally set to cloudscale-csi.

Reply:
146f66109adf9590b31434fdb984430b0f02297b